platform/ios/scripts/package.sh

```diff
@@ -77,8 +76,7 @@ xcodebuild \
     -workspace ./platform/ios/ios.xcworkspace \
     -scheme ${SCHEME} \
     -configuration ${BUILDTYPE} \
-    -sdk iphonesimulator \
-    -jobs ${JOBS} | tee ${LOG_PATH} | xcpretty
+    -sdk iphonesimulator | xcpretty
```

Comment:
Don’t want raw logs? 😢

Reply:
Actually, it looks like CI jobs [will fail](https://circleci.com/gh/mapbox/mapbox-gl-native/188344) if this log doesn’t exist.